samples/net/openthread/coap_server/README.rst

```diff
@@ -20,24 +21,49 @@ Requirements
 User interface
 **************
 
+The following board buttons are used by this sample:
```

Comment:
No need for this sentence, also because you mention buttons, but describe buttons and LEDs. Delete this.